types/migrations/20220518174701_edited_collections.sql

```diff
@@ -0,0 +1,28 @@
+-- Add migration script here
+
+-- A file for collections and chapters
+
+create extension if not exists "ltree";
+
+create table collections (
```

Comment:
Our convention is to use singular table names, so `collection`

Reply:
`collection` seems to be recognized as a data type or something known to sql, so `edited_collection` would be a good choice rather than `collection` to avoid confusion. Will be made singular though.

Reply:
`collection` gets highlighted as a recognized term, and its apparently a data type in sql. To avoid confusion/risk, I think `edited_collection` is a good name.